src/cmd/linuxkit/pkglib/git.go

```diff
@@ -89,6 +89,10 @@ func (g git) treeHash(pkg, commit string) (string, error) {
 		return "", err
 	}
 
+	if len(out) == 0 {
```

Comment:
We generally prefer `out == ""` in this code, but otherwise LGTM.

Reply:
Ah, that's what I wrote at first :)